scss/components/_contact-card.scss

```diff
@@ -44,8 +45,35 @@
   }
 }
 
+.civicase__contact-avatar {
+  background: $contact-avatar-background;
+  padding: 5px;
+}
+
+.civicase__contact-additional__container--avatar {
+  background: $contact-avatar-background;
+  margin-left: 0;
+  margin-top: -10px;
+  opacity: 1 !important;
+  padding: 5px;
+
+  .dropdown-menu {
+    top: 29px;
```

Comment:
We should avoid magic number. Styling of the dropdown should be done generically.

Reply:
I have made some changes to make it 30px. But still the number needs to be here, because we are attaching the arrow, so the menu needs to be pushed down.

Reply:
Can't we have a more generic logic. Like [this](https://github.com/compucorp/uk.co.compucorp.civicase/pull/10/files#diff-f20fb694fca5c66b8b48c2b06ccb9648R28)

Reply:
I really liked that solution, applied the same. Thanks a lot.